packages/upswyng-native/package.json

```diff
@@ -1,72 +1,74 @@
-    "@types/jest": "^24.0.20",
-    "@types/react": "^16.9.2",
-    "@types/react-native": "^0.57.65",
-    "@types/react-native-material-ui": "^1.32.0",
-    "@types/react-router-native": "^4.2.4",
-    "algoliasearch": "^3.35.1",
-    "autoprefixer": "^9.6.4",
-    "babel-preset-expo": "^6.0.0",
-    "jest": "^24.9.0",
-    "node-sass": "^4.12.0",
-    "npm-run-all": "^4.1.5",
-    "ts-jest": "^24.2.0",
-    "typescript": "^3.7.3"
-  },
-  "rnpm": {
-    "assets": [
-      "./assets/fonts"
-    ]
-  }
-}
+    "name": "@upswyng/upswyng-native",
+    "private": true,
+    "license": "MIT",
+    "version": "0.1.0",
+    "main": "__generated__/AppEntry.js",
+    "scripts": {
+        "start": "expo start",
+        "android": "expo start --android",
+        "ios": "expo start --ios",
+        "web": "expo start --web",
+        "eject": "expo eject",
+        "postinstall": "expo-yarn-workspaces postinstall"
+    },
+    "dependencies": {
+        "@upswyng/upswyng-types": "0.1.0",
+        "axios": "^0.19.0",
+        "canonical-instance": "^2.0.0",
+        "compression": "^1.7.1",
+        "cors": "^2.8.5",
+        "debounce": "^1.2.0",
+        "devalue": "^2.0.0",
+        "expo": "^34.0.1",
+        "expo-font": "^6.0.1",
+        "express": "^4.17.1",
+        "express-session": "^1.16.2",
+        "fast-deep-equal": "^2.0.1",
+        "fast-equals": "^2.0.0",
+        "grant": "^4.6.2",
+        "grant-express": "^4.6.2",
+        "grant-oidc": "0.0.3",
+        "moment": "^2.24.0",
+        "mongoose": "^5.6.13",
+        "react": "16.8.3",
+        "react-dom": "16.8.6",
+        "react-native": "https://github.com/expo/react-native/archive/sdk-34.0.0.tar.gz",
+        "react-native-maps": "~0.24.0",
+        "react-native-material-ui": "^1.30.1",
+        "react-native-svg": "~9.5.1",
+        "react-native-vector-icons": "^4.6.0",
+        "react-native-web": "^0.11.4",
+        "react-router-native": "^5.0.1",
+        "rrule": "^2.6.2"
+    },
+    "devDependencies": {
+        "@babel/core": "^7.0.0",
+        "@babel/plugin-syntax-dynamic-import": "^7.0.0",
+        "@babel/plugin-transform-runtime": "^7.0.0",
+        "@babel/preset-env": "^7.0.0",
+        "@babel/runtime": "^7.0.0",
+        "@types/algoliasearch": "^3.34.3",
+        "@types/cli-progress": "^1.8.1",
+        "@types/debounce": "^1.2.0",
+        "@types/jest": "^24.0.20",
+        "@types/react": "^16.9.2",
+        "@types/react-native": "^0.57.65",
+        "@types/react-native-material-ui": "^1.32.0",
+        "@types/react-router-native": "^4.2.4",
+        "algoliasearch": "^3.35.1",
+        "autoprefixer": "^9.6.4",
+        "babel-preset-expo": "^6.0.0",
+        "expo-yarn-workspaces": "^1.2.1",
+        "jest": "^24.9.0",
+        "node-sass": "^4.12.0",
+        "npm-run-all": "^4.1.5",
+        "ts-jest": "^24.2.0",
+        "typescript": "^3.7.3"
+    },
+    "rnpm": {
+        "assets": [
+            "./assets/fonts"
+        ]
+    }
+}
```

Comment:
check this out...https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files